scripts/build/arch/alpha.sh

```diff
@@ -4,3 +4,16 @@ CT_DoArchTupleValues () {
     # The architecture part of the tuple:
```

Comment:
My comment with this patch is specifically with the commit message:
I would change:

  It doesn't seem right having architecture dependent code in something generic like the (e)glibc-common script.

to:

  It doesn't seem right having architecture dependent code a specific libc implementation script.

Sine the (e)glibc-common script no longer exists.